src/app/join/components/JoinCheckboxContainer/JoinCheckbox/JoinCheckbox.tsx

```
@@ -6,10 +6,10 @@ import {
   labelWrapper,
   requiredCheckboxText,
 } from './JoinCheckbox.styles';
-import { tipAlert, tipWrapper } from '../../../JoinInput/JoinInput.styles';
 
 import Icon from '@/components/Icon';
 import theme from '@/styles/theme';
+import { tipAlert, tipWrapper } from '../../JoinInput/JoinInput.styles';
```

ESLint

[error] `../../JoinInput/JoinInput.styles` import should occur before import of `@/components/Icon`

(import/order)